app.py

```diff
@@ -4,9 +4,98 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+todos = [
+    {'id': 1, 'title': 'First Todo', 'description': 'do this tomorrow'},
+    {'id': 2, 'title': 'Second Todo', 'description': 'do this day after'},
+    {'id': 3, 'title': 'Third Todo', 'description': 'do this day after that'}]
+
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return 'Hello World! <a href=http://127.0.0.1:5000/todo> todos</a>'
+
+
+@app.route('/todo', methods=['GET'])
+@app.route('/todo/', methods=['GET'])
+def todolist():
+    return render_template('index.html', todos=todos)
```

Comment:
You are meant to display a persistent list of todos (stored in a cookie) by using the existing code in session_items.py.
Currently turning the server off and on again loses all your data because it's just stored in memory.

This should be `todos=session.get_items()` and you should get rid of the `todos` list in this file.
Similarly, adding or deleting todos should make their changes via `session`, which is the name given to the import of session_items.py